### ISSUE TITLE
iOS13 simulator with xCode11

### ISSUE THREAD

**Issue by mohammadou1** [expo/expo#5721]
Hello, I've someone posting this issue before when iOS13 was on beta, their solution was to revert back to iOS12.

but now since my xCode upgraded to 11 and simulators only supporting iOS13 :(
i have no idea why am facing this issue, when I use expo start or npm start on the terminal

expo installs on the simulator,
but it keeps saying expo is being installed without installing the app and gets stuck like the image shows.

tried to upgrade sdk to 35, tried to uninstall reinstall expo on the simulator

<img width="337" alt="Screen Shot 2019-09-21 at 2 16 22 PM" src="https://user-images.githubusercontent.com/47993969/65373043-8feb5500-dc80-11e9-8ce1-d866f9a29659.png">
<img width="452" alt="Screen Shot 2019-09-21 at 2 15 52 PM" src="https://user-images.githubusercontent.com/47993969/65373044-8feb5500-dc80-11e9-93f2-639f0ae3d96e.png">

**Comment by mzgnr** [comment]
@mohammadou1 Quick workaround is copy the url starts with exp:// expo application will show it on the clipboard section and click it

**Comment by JordanLauriol** [comment]
i have same problem..

**Comment by tsapeta** [comment]
What's the version of your `expo-cli`? You can check it via `expo --version`.

**Comment by Craft4u** [comment]
> What's the version of your `expo-cli`? You can check it via `expo --version`.

I have same problem. 
expo-cli 3.1.0 version

**Comment by mohammadou1** [comment]
> What's the version of your `expo-cli`? You can check it via `expo --version`.

it's 3.0.10

**Comment by mohammadou1** [comment]
> @mohammadou1 Quick workaround is copy the url starts with exp:// expo application will show it on the clipboard section and click it

this actually works thank you, but is there any news about a major fix for this issue?

**Comment by iRoachie** [comment]
@tsapeta Same here.

expo version: 3.1.0
mac os version: 10.14.6 (mojave)
xcode version: 11.0 (11A420a)

Things I have tried:
```
yarn start
"press i"
```
---

```
yarn start -i
```
---
```
yarn start
# Open Expo DevTools
# Select Run on iOS Simulator
```

All the same results. It seems that expo is unable to get the system event that expo installer has completed installation or that the expo client already exists on the simulator. It will always attempt to install the Expo client app, regardless if it's already installed.

Android is unaffected.

**Comment by iRoachie** [comment]
Here are some logs from the cli when I attempted to close the process:

```
Error running `xcrun simctl list devices --json`:
Error installing or running app. Error: xcrun exited with signal: SIGINT
Error running `xcrun simctl list devices --json`:

Error installing or running app. Error: xcrun exited with signal: SIGINT
setRawMode EIO
```

**Comment by tsapeta** [comment]
Thank you guys for all the details you gave me 😉 
I'm working on it right now, the fix should be out soon - I will let you know here.

**Comment by tsapeta** [comment]
It's been fixed in this PR: https://github.com/expo/expo-cli/pull/1018 and published in `expo-cli@3.1.1` so you just need to update it to the latest version 😉
